docs/docs/30-administration/22-backends/50-nixos.md

```diff
@@ -0,0 +1,88 @@
+# NixOS Deployment
```

Comment:
NixOS is not a backend?!

Reply:
Not really you deploy the go binary and utilize a backend like docker, local or anything else.

Reply:
In the added example we use the docker backend

Reply:
did I mess something up or do you have improvements?